### PR TITLE
Minor updates to deck UI

### PR DESCRIPTION
**Pull request by qhuynh96**
* Changes state indicator to squares where states can be differentiated by colors and adds tooltips to the indicator.
* Fixes the problem that Roboto font is not displayed properly.
* Increases font size
* Uses material design icons for Prow Job page.
minor

### CODE REVIEW

**Comment by BenTheElder** [comment]
Nice! I'm in favor of this change.
for context though: the existing red/green/yellow icons were there to mimic GitHub.

File: prow/cmd/deck/static/command-help.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <link id="favicon" rel="icon" type="image/png" href="favicon.ico">
         <link rel="stylesheet" type="text/css" href="style.css">
         <link rel="stylesheet" type="text/css" href="extensions/style.css">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
```
Comment (BenTheElder) [inline prow/cmd/deck/static/command-help.html]: this is wrong, we need to load the font weights for bold and normal.

Comment (BenTheElder) [reply prow/cmd/deck/static/command-help.html]: same below, I added this intentionally :-)

Comment (qhuynh96) [reply prow/cmd/deck/static/command-help.html]: ack

**Comment by qhuynh96** [comment]
@BenTheElder I revert the font loading changes.

**Pull request merged**